packages/core-components/src/layout/InfoCard/InfoCard.tsx

```diff
@@ -92,6 +92,10 @@ const VARIANT_STYLES = {
       flexDirection: 'column',
       height: 'calc(100% - 10px)', // for pages without content header
       marginBottom: '10px',
+      breakInside: 'avoid',
```

Comment:
Hm this is NOT inside the media selector. That risks affecting non-print use cases right?

Reply:
no, else the rendering will not taken into account, changed into avoid page only a page concept lives in print

Reply:
I could not run the techdocs locally btw, so if you could help me then i can also validate this. I run docker desktop but it still failed...